cmd/kubeadm/app/apis/kubeadm/v1beta1/doc.go

```diff
@@ -202,7 +202,7 @@ limitations under the License.
 // 	controlPlaneEndpoint: "10.100.0.1:6443"
 // 	apiServerExtraArgs:
 // 	  authorization-mode: "Node,RBAC"
-// 	controlManagerExtraArgs:
+// 	controllerManagerExtraArgs :
```

Comment:
[1]

Reply:
Updated. thanks

Reply:
hm the space before ":" seems to be here still.